Clear/ExecLemmas.lean

```diff
@@ -24,13 +24,13 @@ variable {s s' : State}
 -- ============================================================================
 
 -- | Executing a continue is the same as setting the `jump` field to `Continue`.
-lemma Continue' : exec fuel .Continue s = 🔁 s := rfl
+lemma Continue' : exec fuel .Continue s = 🔁 s := by unfold exec; rfl
 
 -- | Executing a break is the same as setting the `jump` field to `Break`.
-lemma Break' : exec fuel .Break s = 💔 s := rfl
+lemma Break' : exec fuel .Break s = 💔 s := by unfold exec; rfl
 
 -- | Executing a `Leave` is the same as setting the `jump` field to `Leave`.
-lemma Leave' : exec fuel .Leave s = 🚪 s := rfl
+lemma Leave' : exec fuel .Leave s = 🚪 s := by unfold exec; rfl
```

Comment:
Can you try to put these lemmas into a separate section in which you do `unseal exec` and then the proofs should remain `rfl` rather than `unfold exec; rfl`.

Reply:
Hi, I have proposed a fix in https://github.com/NethermindEth/Clear/pull/5.
- [Eliminate need for adding `unfold exec/eval/evalArgs;`](https://github.com/NethermindEth/Clear/commit/e3ea4f655df7c129181b89c5e3bc660831c80d47)

This has been done for all similar suggestions to this.